src/ch02/write_to_any_slot.md

```diff
@@ -1,6 +1,6 @@
 # Writing to any storage slot
 
-On Starknet, a contract's storage is a map with 2^251 slots, where each slot is a felt which is initialized to 0.
+On Starknet, a contract's storage is a map with \\( 2^{251} \\) slots, where each slot is a `felt252` which is initialized to 0.
```

Comment:
Make it look like: 
![image](https://github.com/NethermindEth/StarknetByExample/assets/19427053/02e0d20c-2414-4745-a94f-9a9b025bc533)